src/bentoml/_internal/service/service.py

```diff
@@ -30,12 +31,13 @@
     from ..bento import Bento
     from ...grpc.v1 import service_pb2_grpc as services
     from .openapi.specification import OpenAPISpecification
-
 else:
     grpc, _ = import_grpc()
 
 logger = logging.getLogger(__name__)
 
+SerializationStrategy = t.Literal["EXPORT_BENTO", "LOCAL_BENTO", "REMOTE_BENTO"]
```

Comment:
We should make this an enum 😄

Reply:
I slightly prefer using string literal for this case